blueos_repository/registry.py

```diff
@@ -57,11 +58,52 @@ async def fetch_remote_tags(self, repository: str) -> List[str]:
                 print(valid_images)
                 return valid_images
 
+    def get_current_arch(self) -> str:
+        """Maps platform.machine() outputs to docker architectures"""
+        arch_map = {"x86_64": "amd64", "aarch64": "arm", "armv7l": "arm"}  # TODO: differentiate arm64v8/ arm32v7
+        machine = platform.machine()
```

Comment:
I'm not sure if I follow, if the registry is used on CI, why does it matter the architecture that it's running ?

Reply:
lol, good point. It doesn't really.